### ISSUE TITLE
Brave-specific JS errors appearing on brave://adblock

### ISSUE THREAD

**Issue by pes10k** [brave/brave-browser#5182]
Error:
```
[Deprecation] HTML Imports is deprecated and will be removed in M73, around March 2019. Please use ES modules instead. See https://www.chromestatus.com/features/5144752345317376 for more details.
VM14:1 Uncaught ReferenceError: brave_new_tab is not defined
    at <anonymous>:1:1
VM15:1 Uncaught ReferenceError: brave_adblock is not defined
    at <anonymous>:1:1
```

Doesn't seem to happen on reload, only on first open.  Doesn't seem to break things, page seems correct, but probably not ideal ;)

**Comment by snyderp** [comment]
@bbondy If you can point me to where this is implemented in the browser, im happy to give it a go, while Im waiting for PageGraph builds to complete…

Also, if its not a _huge_ additional lift, i could also work on making brave://adblock styled like brave://settings/ (if someone could hand hold me on the basics, assuming its mostly just getting the markup / styles correct…)

**Comment by isiahmeadows** [comment]
I see this also when opening a new tab. I decided to take a look at the source in both GitHub and in the devtools, and I found three culprits in a very small dependency tree:

- [Here, in Brave's new tab code](https://github.com/brave/brave-core/blob/master/components/brave_new_tab_ui/brave_new_tab.html#L8)
- [Here, in Chrome's `chrome://resources/html/cr.html`](https://cs.chromium.org/chromium/src/ui/webui/resources/html/event_tracker.html?g=0&l=1)
- [Here, in Chrome's `chrome://resources/html/promise_resolver.html`](https://cs.chromium.org/chromium/src/ui/webui/resources/html/promise_resolver.html?g=0&l=1)
- And that imports [this file, Chrome's `chrome://resources/html/assert.html`](https://cs.chromium.org/chromium/src/ui/webui/resources/html/assert.html?g=0), which does not import anything.

Chrome [has a bug on this](https://bugs.chromium.org/p/chromium/issues/detail?id=937747) related to this, closed as a duplicate of [this bug](https://bugs.chromium.org/p/chromium/issues/detail?id=937746), and they are almost certainly blocked on that bug.

**Comment by bsclifton** [comment]
Still happens on latest, just a different date/Chromium milestone 😂 
```
[Deprecation] HTML Imports is deprecated and will be removed in M80, around February 2020. Please use ES modules instead. See https://www.chromestatus.com/features/5144752345317376 and https://developers.google.com/web/updates/2019/07/web-components-time-to-upgrade for more details.
```

**Comment by antonok-edm** [comment]
I'm no longer seeing this error. I do see a [separate issue](https://github.com/brave/brave-browser/issues/14752), but I'll close this one as it should have been resolved already (we are long past the deprecation date).
